src/CLI/Check.php

```diff
@@ -96,7 +96,7 @@ protected function printSummaryLine(OutputInterface $output, int $assertionCount
 
     protected function printHeadingLine(OutputInterface $output): void
     {
-        $output->writeln("<info>PHPArkitect 0.0.1</info>\n");
+        $output->writeln("<info>PHPArkitect</info>\n");
```

Comment:
What is the idea behind this change? Imo we should keep it and adding some more infos (a la composer).

Reply:
Yes, we can do this.
My thought was to do a little step because now it's wrong that information.
We can after adding to composer.json the version information.

Or do you prefer to do this in a single PR?

Reply:
@micheleorselli I am watching other tools and it seems that no one takes the version from other parts.

Example from PHPUnit

https://github.com/sebastianbergmann/phpunit/blob/master/src/Runner/Version.php

They have hardcoded the version and when. they prepare a new release they change that string.

We can keep our code but we need to change it with the next release.
Previously I thought that was a bad practice but now I don't think so.

What do you think?

Reply:
https://stackoverflow.com/questions/50621578/get-version-from-composer-json-symfony/50644633
I think having the version just in one place is a good idea, and composer.json seems the most obvious one..

Reply:
+1 for using composer

Reply:
I have seen that link but it's useful for installed packages not for a custom parameter I think.
Or am I wrong?

@micheleorselli @fain182

Reply:
I have taken the version from our composer.json file what do you think @micheleorselli @fain182?
Is it good enough?

Reply:
Simple and effective 👏